lib/util/host-rules.spec.ts

```diff
@@ -237,6 +237,21 @@ describe('util/host-rules', () => {
         })
       ).toEqual({ token: 'abc' });
     });
+    it('matches shortest matchHost first', () => {
```

Comment:
Shouldn't the description say "longest" instead of "shortest"?

Reply:
No, it matches shortest first so that the last matched takes precedence (it matches all)